src/app/models/projectNotification.ts

```diff
@@ -0,0 +1,50 @@
+import { CommentPeriod } from "./commentperiod";
+
+export class ProjectNotification {
+  _id: string;
+  name: String;
+  type: String;
+  subType: String;
+  nature: String;
+  region: String;
+  location: String;
+  decision: String;
+  decisionDate: Date;
+  description: String;
+  trigger: String;
+  centroid: Array<number>;
+  commentPeriod: CommentPeriod;
+
+  read: Array<String> = [];
+
+ /**
+ * Maps a project notification API response to the data model.
+ *
+ * @param responseData Project Notification API response.
```

Comment:
I know I initially had this as I thought I was going to have to do some mapping, but turns out I didn't This can be removed.

Reply:
Ok, i'll copy the latest copy of the model